hikari/impl/entity_factory.py

```diff
@@ -1084,10 +1078,7 @@ def deserialize_guild_member_ban(self, payload: data_binding.JSONObject) -> guil
     def _set_partial_guild_attributes(payload: data_binding.JSONObject) -> _PartialGuildFields:
         features = []
         for feature in payload["features"]:
-            try:
-                features.append(guild_models.GuildFeature(feature))
-            except ValueError:
-                features.append(feature)
+            features.append(guild_models.GuildFeature(feature))
```

Comment:
For guild.features we were likely typing them as `Union[Enum, str]` on the model, I don't see that being removed anywhere and unless I missed it so would the Union with str now be unnecessary?

Reply:
Same as above